src/demo-app/a11y/input/input-a11y.html

```diff
@@ -0,0 +1,56 @@
+<section>
+  <h2>Basic input box (e.g. name field)</h2>
+  <md-input-container floatPlaceholder="never">
+    <input mdInput placeholder="First name" [(ngModel)]="firstName">
```

Comment:
Not needed when you specify a placeholder since it gets placed inside of a `<label>` tag